run.sh

```diff
@@ -7,7 +7,7 @@ declare DEFAULT_DEVICE="--auto"
 # - the green LED    (1 << 14)
 # - an infra-red LED (1 <<  2)
 # Override with `LED_MASK=`.
-declare DEFAULT_LED_MASK=0x4004
+declare DEFAULT_LED_MASK=0xF000
```

Comment:
Why has this changed?

Reply:
just easier to see if the flash loop is working, by enabling part of the visible LEDs

Reply:
hmm, I used to prefix it with a "tmp:" in commit message, probably better to keep it?